### ISSUE TITLE
Spotify crashes after a few seconds of opening after using the patch

### ISSUE THREAD

**Issue by qw1642190428** [mrpond/BlockTheSpot#557]
Due to network issues, it couldn't be installed automatically, so I chose to install it manually. It worked fine at first, but it started crashing on the second or third day. After removing the patch, it started working fine again. If I want to use the patch, I need to reinstall Spotify. After reinstalling and using the patch, it started crashing again two days later.

**Comment by Miky172** [comment]
Same issue here, BlockTheSpot leads Spotify to crash.
But there is something more I noticed, I tried disabling and enabling one by one all BlockTheSpot features in the config.ini file and I'm pretty sure that it's "Block_Banner" the issue.
While it is set to "1" Spotify crashes, but it does not while the value is "0".

**Comment by LyPedeuh** [comment]
> Same issue here, BlockTheSpot leads Spotify to crash. But there is something more I noticed, I tried disabling and enabling one by one all BlockTheSpot features in the config.ini file and I'm pretty sure that it's "Block_Banner" the issue. While it is set to "1" Spotify crashes, but it does not while the value is "0".

Yep, it was working fine until today, i changed the value of "Block_Banner" and it works fine again.

**Comment by gcavalcantes** [comment]
Had the same problem happen to me just now.

> Same issue here, BlockTheSpot leads Spotify to crash. But there is something more I noticed, I tried disabling and enabling one by one all BlockTheSpot features in the config.ini file and I'm pretty sure that it's "Block_Banner" the issue. While it is set to "1" Spotify crashes, but it does not while the value is "0".

Can confirm that changing "Block_Banner" value to 0 fixed the issue.

**Comment by ModsAreFlags** [comment]
> Same issue here, BlockTheSpot leads Spotify to crash. But there is something more I noticed, I tried disabling and enabling one by one all BlockTheSpot features in the config.ini file and I'm pretty sure that it's "Block_Banner" the issue. While it is set to "1" Spotify crashes, but it does not while the value is "0".

Also worked for me! Thanks so much.

**Comment by mrpond** [comment]
Hi, I can't reproduce the issue.
try use problem report form

also this is my version at the moment.
```
Spotify for Windows (64 bit)
1.2.45.454.gc16ec9f6
xpui_2024-08-26_1724674643080_c16ec9f
cef_127.3.4+ge9e2e14+chromium-127.0.6533.100
Runtime: Chrome
Build System: Bazel
```
